qemu/tests/boot_with_different_vectors.py

```diff
@@ -121,7 +107,6 @@ def check_interrupt(session, vectors):
         if int(vectors) < 0:
             continue
         session = vm.wait_for_login(timeout=login_timeout)
-        enable_multi_queues(vm)
```

Comment:
What about the old versions?

Reply:
no RHEL.3 RHEL.4 RHEL.5 RHEL.6
the patch in which changed the behavior introduced since RHEL7.5
so i think we can simply remove the configuration of mq because it will become a general behavior

Reply:
So I'd like to know how is compatibility for guests prior to RHEL.7.5, does `no RHEL.7.0, .., RHEL.7.4` is needed in the cfg file?

Reply:
For this consideration, I think I will need to disable RHEL.7.0 - 7.4